orbit/pkg/table/ioreg/ioreg.go

```diff
@@ -136,7 +134,6 @@ func (t *Table) generate(ctx context.Context, queryContext table.QueryContext) (
 
 func (t *Table) flattenOutput(dataQuery string, systemOutput []byte) ([]dataflatten.Row, error) {
 	flattenOpts := []dataflatten.FlattenOpts{
-		dataflatten.WithLogger(t.logger),
```

Comment:
Is there a reason we no longer wish to add a logger to the flattener?

Reply:
Nope, it should be there. Removed it by accident.